src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs

```diff
@@ -80,7 +82,9 @@ internal float WorldPositionY
             get
             {
                 float temp = 0.0f;
-                GetProperty(View.Property.WORLD_POSITION_Y).Get(out temp);
+                PropertyValue _wordPositionY = GetProperty(View.Property.WORLD_POSITION_Y);
```

Comment:
could you check this NUI ground rule suggestion #2 ? (https://code.sec.samsung.net/confluence/display/GFX/NUI+Ground+Rule)
How do you think about using camel case for private field's naming.

Reply:
Done